vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -1405,6 +1524,99 @@ CreativeAdNotificationList AdsImpl::GetAdsForUnseenAdvertisers(
   return unseen_ads;
 }
 
+CreativePublisherAdList AdsImpl::GetEligiblePublisherAds(
```

Comment:
nit: Given `GetUnseenPublisherAdsAndRoundRobinIfNeeded` below, should this be

`GetEligiblePublisherAdsAndRoundRobinIfNeeded`

?

Reply:
Refactored but these names are correct